app/src/main/java/com/example/fitnessconstructor/ui/MainActivity.kt

```diff
@@ -38,14 +55,32 @@ class MainActivity : AppCompatActivity() {
 
         // Setup the ActionBar with navController and 3 top level destinations
         appBarConfiguration = AppBarConfiguration(
-            setOf(R.id.workoutFragment, R.id.statisticFragment,  R.id.appSettingsFragment)
+            setOf(R.id.workoutFragment, R.id.statisticFragment, R.id.appSettingsFragment)
         )
         setupActionBarWithNavController(navController, appBarConfiguration)
+    }
 
-        //remove action bar while launch app
-        supportActionBar?.hide()
-
+    private fun splashScreenLaunch() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+            val splashScreen = installSplashScreen()
```

Comment:
You can use annotations for more readable code and for Lint check, such as: `@TargetApi(Build.VERSION_CODES.S)` or `@RequiresApi(api = Build.VERSION_CODES.S)`

For example:
```
override fun onCreate(savedInstanceState: Bundle?) {
    ...
    splashScreenLaunch()
    ...
}

@TargetApi(Build.VERSION_CODES.S)
private fun splashScreenLaunch() {
    if (isApiHigher31()) {
    ...
    }
}

fun isApiHigher31() = Build.VERSION.SDK_INT >= Build.VERSION_CODES.S
```
or
```
override fun onCreate(savedInstanceState: Bundle?) {
    ...
    if (isApiHigher31()) splashScreenLaunch()
    ...
}

@RequiresApi(api = Build.VERSION_CODES.S)
private fun splashScreenLaunch() {
...
}

fun isApiHigher31() = Build.VERSION.SDK_INT >= Build.VERSION_CODES.S
```

Reply:
OK! Thanks!